### PR TITLE
Update README.rst (grammar)

### PR DESCRIPTION
**Pull request by ArjunSharda**
The grammar in the README.rst are bad. I tried updating them in my fork and I made a PR for this.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -3,15 +3,15 @@ Linux Integration Services Automation (LISA)
 
 |CI Workflow| |GitHub license| |Docs|
 
-**Linux Integration Services Automation (LISA)** is a Linux quality
+**Linux Integration Services Automation (LISA)** is an Linux quality
```
Comment (squirrelsc) [inline README.rst]: Is it a right change?

Comment (ArjunSharda) [reply README.rst]: Oopsie, accident. My bad

Comment (ArjunSharda) [reply README.rst]: Updated, thank you.

**Comment by ArjunSharda** [comment]
Could you review this when you have time? @squirrelsc

**Review by LiliDeng — APPROVED** [review]

**Review by squirrelsc — APPROVED** [review]

**Pull request merged**